dev-libs/zziplib/zziplib-0.13.69.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit libtool eutils flag-o-matic python-any-r1
+
+DESCRIPTION="Lightweight library for extracting data from files archived in a single zip file"
+HOMEPAGE="https://github.com/gdraheim/zziplib"
+SRC_URI="https://github.com/gdraheim/zziplib/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( LGPL-2.1 MPL-1.1 )"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE="doc sdl static-libs test"
+
+RDEPEND="
+	sys-libs/zlib
+	sdl? ( >=media-libs/libsdl-1.2.6 )"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	virtual/pkgconfig
+	test? ( app-arch/zip )"
+
+src_prepare() {
+	python_fix_shebang .
+
+	# workaround AX_CREATE_PKGCONFIG_INFO bug #353195
+	sed -i \
+		-e '/ax_create_pkgconfig_ldflags/s:$LDFLAGS::' \
+		-e '/ax_create_pkgconfig_cppflags/s:$CPPFLAGS::' \
+		configure || die
+
+	# zziplib tries to install backwards compat symlinks we dont want
+	sed -i -e '/^zzip-postinstall:/s|$|\ndisable-this:|' Makefile.in || die
+	sed -i -e '/^install-exec-hook:/s|$|\ndisable-this:|' zzip/Makefile.in || die
+
+	elibtoolize
+
+	# Do an out-of-tree build as their configure will do it automatically
+	# otherwise and that can lead to funky errors. #492816
+	mkdir -p build
+
+	eapply_user
+}
+
+src_configure() {
+	cd "${S}"/build
+
+	append-flags -fno-strict-aliasing # bug reported upstream
+	export ac_cv_path_XMLTO= # man pages are bundled in .tar's
+
+	# Disable aclocal probing as the default path works #449156
+	ECONF_SOURCE=${S} \
+	ACLOCAL=true \
+	econf \
+		$(use_enable sdl) \
+		$(use_enable static-libs static)
+	MAKEOPTS+=' -C build'
+}
+
+src_test() {
+	# need this because `make test` will always return true
+	# tests fail with -j > 1 (bug #241186)
+	emake -j1 check
```

Comment:
That's weird, either with or without that I always get that its not enabled:
`>>> Test phase [not enabled]: dev-libs/zziplib-0.13.69`

Reply:
Did you try with `FEATURES="test"`? Anyway the tests fail horribly because they're trying to download bunch of different .zip files from the internet. I think you're better off adding `RESTRICT="test"` in this ebuild, because I've never seen 
```
SRC_URI="test? ( 
	testfile1.zip
	testfile2.zip
	testfile3.zip
	testfile4.zip
)
```

Reply:
Yeah, it doesn't seem to work correctly following the src_test guidelines. I've removed the test flag and dependencies (since they were only required for this phase) and set the test to restricted.